src/generators/default_map_generator_job.cpp

```diff
@@ -105,7 +302,7 @@ height_map default_map_generator_job::generate_height_map(size_t width, size_t h
 
 	for(size_t i = 0; i != iterations; ++i) {
 
-		// (x1,y1) is the location of the hill,
+		// (x1,y1) is the map_location of the hill,
```

Comment:
(On a side note, things like this is the reason why I often use "Replace & Find Next" repeatedly, rather than "Replace All", even when I think I want to replace all instances. Though XCode actually allows me to batch replace any subset of the results if I use the whole-project search.)